packages/cli/src/core/plugin.js

```diff
@@ -6,6 +6,22 @@ const HttpMock = require("./http-mock");
 const Collection = require("./collection");
 const path = require("path");
 const Coverage = require("./coverage");
+const {getPackageJson} = require("../utils/fs");
+
+function formatCommand(command) {
+  if (typeof command !== "string") {
+    throw new Error(`Plugin: command should be a string but got: ${command}`);
+  }
+
+  const isNpmCommand = command.startsWith("npm run");
```

Comment:
I think we need to be careful because we might have user that just use the command `npm start`

Reply:
Yeah, If the user chooses to write it manually. My current code is only compliant with what we automate (as we always add `npm run`)